### ISSUE TITLE
Exporting to PNG and embedding the scene crashes Chromium

### ISSUE THREAD

**Issue by nrutman** [excalidraw/excalidraw#6349]
Multiple people on my team (using various Chromium browsers: Chrome, Arc, etc.) have found that exporting a PNG and embedding the scene completely crashes the browser.

**Comment by bhb603** [comment]
Same here...
It only crashes when "Embed scene" is selected.
It crashes for PNG and SVG exports with scened embedding for me on Chrome + Arc.

**Comment by alexforster** [comment]
I'm seeing this too, on regular, stable Chrome.

**Comment by dwelle** [comment]
Hello. If anyone can still reproduce, what is your OS and Chrome version?

Does the crash happen on all drawings, or just bigger ones? If on larger ones, what are the dimensions & size? You can check by right-clicking on canvas and opening the "stats for nerds":

![image](https://user-images.githubusercontent.com/5153846/225885183-169079b2-cbdc-49c5-9633-b446134f7084.png)

**Comment by namoscato** [comment]
This is still occurring for me on macOS Chrome v111.0.5563.64 (Official Build) (arm64) on all drawings, i.e. this small one:

<img width="137" alt="Screenshot 2023-03-17 at 6 55 53 AM" src="https://user-images.githubusercontent.com/847532/225886303-b0aa257a-ef79-4209-b53e-0adb110b99d1.png">

**Comment by bhb603** [comment]
Still crashing... even with very tiny drawing:

![e716ef3b3646bca325fc1fba72ecf57e](https://user-images.githubusercontent.com/4773480/225919786-896efad3-84e6-4197-8b5d-abdc3ec8d30f.png)

Browser:
```
Arc Version 0.94.0 (37738)
Chromium Engine Version 111.0.5563.64
```

**Comment by dwelle** [comment]
Strange.

@namoscato @bhb603 both of you have 111.0.5563.64, could this be it? I have 111.0.5563.65, and I tried to find 111.0.5563.64 build, but couldn't find it in https://commondatastorage.googleapis.com/chromium-browser-snapshots/index.html (best I could do is 111.0.5563.0, which still works).

Could you try update if it allows you, and if not, try one of the newer channels (Canary, Beta, Dev) if it's fixed for you?

**Comment by namoscato** [comment]
Yeah, as noted in https://github.com/excalidraw/excalidraw/issues/6349#issuecomment-1473580678, it works for me in v113.0.5656.0 (canary).

**Comment by dwelle** [comment]
Oh, I missed that was you. Alright, so the working theory is this is a browser issue fixed in newer versions. We'll keep this open for a while and close unless new info is discovered.

**Comment by aryyh** [comment]
I'm having this issue too, also on 111.0.5563.64
```
Google Chrome | 111.0.5563.64 (Official Build) (arm64)
Revision | c710e93d5b63b7095afe8c2c17df34408078439d-refs/branch-heads/5563@{#995}
OS | macOS Version 12.6.3 (Build 21G419)
JavaScript | V8 11.1.277.13
User agent | Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/111.0.0.0 Safari/537.3
```
Seems 111.0.5563.6**5** isn't available for me yet through the usual google chrome updates, hopefully i'll be to able to report back too that it's working, once I get that build.

**Comment by noamnelke** [comment]
I'm also on `Version 111.0.5563.64 (Official Build) (arm64)`, experiencing the issue since updating to this version and don't have a different version available to me on the stable channel.

@dwelle notice that this is an arm64 version (I'm on an m1 mac). If you're on x64 (Intel) it could explain why you don't see this version and others ONLY see this version.

Anyway, frustrating as it is to just wait, the issue seems specific to this version of Chrome, and based on people reporting - canary fixes it, so it may be worth just waiting it out.

If anyone does want to research this and needs data from someone who can reproduce (not sure if there's a way to write logs that survive a browser crash)-- let me know, I'd be happy to help.

As already reported, this happens consistently even with a fresh drawing of a single line, when exporting to an image with scene data embedded.
